honeybee_doe2/writer.py

```diff
@@ -15,8 +15,21 @@
 from honeybee.typing import clean_string
 
 
-def model_to_inp(hb_model):
+def model_to_inp(hb_model, hvac_grouping):
```

Comment:
Let's put a default value here for hvac_grouping so it doesn't break the existing codes. It should be set to `story`

Reply:
done! [see commit](https://github.com/ladybug-tools/honeybee-doe2/pull/59/commits/f17cf169dad9f6b3757eb7a1e0b47837f59495a1)